sdk/ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTest.scala

```diff
@@ -227,6 +227,50 @@ abstract class HttpServiceIntegrationTest
       } yield result should ===("Bob invoked IIou.Transfer")
     }
 
+    // This is a hacky way to determine the package id of a dar,
+    // but might suffice for tests for now.
+    def packageIdOfDar(dar: File): Future[String] = Future {
+      import java.util.zip.ZipInputStream
+      val zis = new ZipInputStream(new java.io.FileInputStream(dar))
+      val entry = zis.getNextEntry()
+      // Paths are formatted as: {packageName}-{packageVersion}-{packageId}/{dalfFile}
+      val pkgId = entry.getName.split("/")(0).split("-")(2)
+      zis.closeEntry()
+      zis.close()
+      pkgId
+    }
+
+    def postCreate(
```

Comment:
I don't think you need to roll your own barebone postCreate, you could use the normal postCreateCommand with an iouCommand, where you can override the normally absent package id part of the template id with the one you have with .copy, no?

Reply:
Again, it's not clear from this PR alone, but I'm planning to use this to create contracts of a different template and package.